Formula/bdw-gc.rb

```diff
@@ -1,20 +1,9 @@
 class BdwGc < Formula
   desc "Garbage collector for C and C++"
   homepage "https://www.hboehm.info/gc/"
+  url "https://github.com/ivmai/bdwgc/releases/download/v8.0.6/gc-8.0.6.tar.gz"
+  sha256 "3b4914abc9fa76593596773e4da671d7ed4d5390e3d46fbf2e5f155e121bea11"
   license "MIT"
-  revision 2
-
-  stable do
-    url "https://github.com/ivmai/bdwgc/releases/download/v8.0.4/gc-8.0.4.tar.gz"
-    sha256 "436a0ddc67b1ac0b0405b61a9675bca9e075c8156f4debd1d06f3a56c7cd289d"
-
-    # Extension to handle multithreading
-    # https://github.com/ivmai/bdwgc/pull/277
-    patch do
-      url "https://github.com/ivmai/bdwgc/commit/5668de71107022a316ee967162bc16c10754b9ce.patch?full_index=1"
-      sha256 "5c42d4b37cf4997bb6af3f9b00f5513644e1287c322607dc980a1955a09246e3"
-    end
```

Comment:
By the way, did you check that the patch has been included in the new release? It seems like it hasn't.

Reply:
Yes, I built it locally from source :)

Reply:
The patch implements a new feature used by a different formula, so it will build from source whether it's in the new release or not.

Reply:
Oh, sorry I did not realise -- perhaps wait for the CI runs to pass, then see if we need to add it back?